src/components/Events/Fall_GM/Fall_GM.js

```diff
@@ -5,57 +5,196 @@ import Countdown from 'react-countdown';
+						<div className="click-to-join">
+							<a href="https://bit.ly/ACMdiscord" target="_blank" rel="noopener noreferrer">
+								<div id="discord-join-link-wrapper" className="join-links">
+									<img src={process.env.PUBLIC_URL + "/images/DiscordLogoWordmark.png"} alt="Discord logo" className="join-links-img"/>
+								</div>
+							</a>
+							<p className="join-us">Join us from 7:10 p.m. PDT onwards in Discord</p>
 						</div>
-						<div className="countdown-cards">
-							<div className="square-background"/>
-							<div className="countdown-numbers">{minutes}</div>
-							<div className="countdown-labels">{minuteString}</div>
+					</div>
+					<div id="gm-program-wrapper">
+					<h2>Program</h2>
+					<div className="row">
+						<p className="timestamps">6:35 p.m. PDT</p>
+						<div className="row-description-wrapper">
+							<h3 className="row-description-heading">Rucha Patki: Welcome</h3>
 						</div>
-						<div className="countdown-cards">
-							<div className="square-background"/>
-							<div className="countdown-numbers">{seconds}</div>
-							<div className="countdown-labels">{secondString}</div>
+					</div>
+					<div className="row">
+						<p className="timestamps">6:38 p.m. PDT</p>
+						<div className="row-description-wrapper">
+							<h3 className="row-description-heading">Alumni: Why ACM?</h3>
+							<p className="row-description-subheading">Hear perspectives of alumni on what their experience with ACM at UCLA meant to them</p>
+							<p>Elena Escalas (class of 2020)</p>
+							<p>Kevin Tan (class of 2020)</p>
+						</div>
+					</div>
+					<div className="row">
+						<p className="timestamps">6:40 p.m. PDT</p>
+						<div className="row-description-wrapper">
+							<h3 className="row-description-heading">Committee Presentations</h3>
+							<p className="row-description-subheading">Learn more about each of ACM's eight committees: what they have to offer, and why you should join them</p>
+							{this.renderCommitteeSpeakers()}
+						</div>
+					</div>
+					<div className="row">
+						<p className="timestamps">7:05 p.m. PDT</p>
+						<div className="row-description-wrapper">
+							<h3 className="row-description-heading">Lightning Round: Joining ACM / ACM Board</h3>
+							<p className="row-description-subheading">How to become even more invovled with ACM beyond attending workshops and events</p>
+							<p>Internal: William Chern</p>
+							<p>ACM Dev Team: Austin Keil, Nhi Van</p>
+							<p>ACM Internship Program: Gloria Yang, Sahen Rai</p>
+							<p>External: Katherine Chong</p>
+						</div>
+					</div>
+					<div className="row">
+						<p className="timestamps">7:10 p.m. PDT</p>
+						<div className="row-description-wrapper">
+							<h3 className="row-description-heading">Tabling and Social</h3>
+							<p className="row-description-subheading">Join us in our official ACM Discord for Committee tabling and socials</p>
```

Comment:
Consider adding a link under `ACM Discord` - it's a nice CTA to round out the rest of the page